views/assignments/doAssignment.html

```diff
@@ -31,7 +28,7 @@
 <script type="text/javascript" src="/{{=request.application}}/static/{{=course_id}}/_static/jquery.ba-bbq.min.js"></script>
 <script type="text/javascript" src="/{{=request.application}}/static/{{=course_id}}/_static/jquery.jsPlumb-1.3.10-all-min.js"></script>
 <script type="text/javascript" src="/{{=request.application}}/static/{{=course_id}}/_static/d3.v2.min.js"></script>
-
+<script type="text/javascript" src="{{=URL('static', 'js/selfgrade.js')}}"></script>
```

Comment:
What and where is selfgrade.js ?? I don't see a new file for this added to the PR and I can't find it anywhere in the existing code base.

Should this be conditionally included only for coursera mode or something?  It doesn't seem like it needs to be included for academy mode.